### PR TITLE
Added commands in CI for building docker images when merged

### PR DESCRIPTION
**Pull request by marrts**

### CODE REVIEW

**Comment by marrts** [comment]
Apparently I did something wrong because now the Foxy and Humble CI's aren't even running.

**Comment by marip8** [comment]
> [Invalid workflow file: .github/workflows/jammy_build.yml#L80](https://github.com/tesseract-robotics/tesseract_ros2/actions/runs/5480376555/workflow)
The workflow is not valid. .github/workflows/jammy_build.yml (Line: 80, Col: 17): Unrecognized named-value: 'ROS_DISTRO'. Located at position 1 within expression: ROS_DISTRO

**Comment by marrts** [comment]
> Click on to `Checks -> <Build Name>` to see the error message from jobs that don't run

Thanks! That error makes sense now that I see it.

File: .github/workflows/focal_build.yml [hunk 1]
```diff
@@ -58,5 +61,38 @@ jobs:
...
+        uses: docker/metadata-action@v3
+        with:
+          images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
+          flavor: |
+            latest=false
+            prefix=
+            suffix=
+          tags: |
+            type=ref,event=branch,prefix=${{ ROS_DISTRO }}-
+            type=semver,pattern={{major}}.{{minor}},prefix=${{ ROS_DISTRO }}-
```
Comment (marip8) [inline .github/workflows/focal_build.yml]:
```suggestion
            type=ref,event=branch,prefix=${{ env.ROS_DISTRO }}-
            type=semver,pattern={{major}}.{{minor}},prefix=${{ env.ROS_DISTRO }}-
```

File: .github/workflows/jammy_build.yml [hunk 2]
```diff
@@ -58,5 +61,38 @@ jobs:
...
+        uses: docker/metadata-action@v3
+        with:
+          images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
+          flavor: |
+            latest=false
+            prefix=
+            suffix=
+          tags: |
+            type=ref,event=branch,prefix=${{ ROS_DISTRO }}-
+            type=semver,pattern={{major}}.{{minor}},prefix=${{ ROS_DISTRO }}-
```
Comment (marip8) [inline .github/workflows/jammy_build.yml]:
```suggestion
            type=ref,event=branch,prefix=${{ env.ROS_DISTRO }}-
            type=semver,pattern={{major}}.{{minor}},prefix=${{ env.ROS_DISTRO }}-
```

File: .github/workflows/focal_build.yml (hunk 1 above)
Comment (marrts) [reply .github/workflows/focal_build.yml]: Thanks, I beat you to it by a split second

**Review by marip8 — APPROVED** [review]

**Pull request merged**